scroll-func.js

```diff
@@ -0,0 +1,7 @@
+let scrollBtn = document.querySelector(".back-to-top");
+window.addEventListener("scroll", function(){
+      let position = window.scrollY;
+    if(position < 200){
+      scrollBtn.style.opacity = "0";
+    }else{scrollBtn.style.opacity = "1";}
+})
```

Comment:
Please add a semicolon at the end of this line, to end the `window.addEventListener` function call. Please see my comment on this PR for more general feedback. Thanks!